### ISSUE TITLE
Can AITemplate prepare to support LoRa for stable diffusion?

### ISSUE THREAD

**Issue by sanbuphy** [facebookincubator/AITemplate#571]
Hello, I am using aitemplate and I find its speed really fast. 
I would like to ask if it can support other additional features such as LoRa. Thank you very much.

**Comment by zero-nnkn** [comment]
Hello, I wonder if AITemplate now supports LoRA for the Diffusion model (load LoRAa before or after compile) ?

**Comment by zero-nnkn** [comment]
> > Hello, I wonder if AITemplate now supports LoRA for the Diffusion model (load LoRAa before or after compile) ?
> 
> can you clone the code an do a test?

Yes, I tried the diffusion with LoRA + AI Template ([example](https://github.com/facebookincubator/AITemplate/tree/main/examples/05_stable_diffusion)). 
But the result is like the model without LoRA, it seems AI Template is not supported this feature yet.

**Comment by foocker** [comment]
> > > Hello, I wonder if AITemplate now supports LoRA for the Diffusion model (load LoRAa before or after compile) ?
> > 
> > 
> > can you clone the code an do a test?
> 
> Yes, I tried the diffusion with LoRA + AI Template ([example](https://github.com/facebookincubator/AITemplate/tree/main/examples/05_stable_diffusion)). But the result is like the model without LoRA, it seems AI Template is not supported this feature yet.

i see from trt, it seems the best way is transform the merged weights to trt. at here, i do not konw the logical of  aitemplate. but maybe just merge the weights? i see controlnet in ([example](https://github.com/facebookincubator/AITemplate/tree/main/examples/05_stable_diffusion))
